### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by LPeter1997**
Added runnable examples to a subfolder. The script `run_examples` executes all of them in order.

### CODE REVIEW

File: examples/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Toolchain
+nuget.config
```
Comment (Kuinox) [inline examples/.gitignore:2]: Why do you need a nuget.config ?

File: scripts/install_toolchain.ps1
Comment (Kuinox) [inline scripts/install_toolchain.ps1:1]: Let's add a powershell comment that explain the purpose of the script (and it's usage).

File: examples/.gitignore (hunk 1 above)
Comment (LPeter1997) [reply examples/.gitignore:2]: Because you can install the toolset right into this folder which creates a nuget.config

File: scripts/install_toolchain.ps1
Comment (LPeter1997) [reply scripts/install_toolchain.ps1:1]: Added a header comment to all scripts

**Review by Kuinox — APPROVED** [review]

**Pull request merged**